css/workflow-visualizer.styl

```diff
@@ -0,0 +1,130 @@
+@import "jsPlumb"
+@import "react-resizable"
+
+.answer-item
+  padding-left: 15px
+  padding-right: 15px
+  margin-top: 0.5em
+  margin-bottom: 0.5em
+  clear: both
+
+.box,
+.box-end
+  min-width: 200px
+  max-width: 400px
+  width: 220px
+  background-color: #ccc
+  border-style: solid
+  border-width: 1px
+  border-radius: 15px
+  position: absolute
+  overflow: hidden
+
+.box
+  resize: horizontal
+  overflow: auto
+  border-bottom-right-radius: 0
+
+.box-head
+  pointer-events: none
+  color: #fff
+  opacity: 0.7
+  float: left
+  margin-left: 10px
+  text-weight: 500
+
+.drag-handel
+  padding-bottom: 10px
+  cursor:  move
+  border-top-left-radius: 15px
+  border-top-right-radius: 15px
+  background-color: #000
+  background-color: rgba(0,0,0,.3)
```

Comment:
rgba syntax is only not supported in IE < 9, which we don't support anyway. Can remove duplicate entries like this.